### DISCUSSION TITLE
访问首页提示500，访问/ui/register时也是500，/1.1/classes/ 404

### DISCUSSION THREAD

**Discussion by Hongwing** [walinejs/waline#1926] [Q&A] [answered]
{"errno":500,"errmsg":"undefined [404 GET blog-discussions.vercel.app/1.1/classes/Comment]"}

server_url: https://blog-discussions.vercel.app/

**Comment by lizheming** [comment, marked as answer]
https://github.com/walinejs/waline/issues/1361

**Reply by Hongwing** [reply]
thx
